insights/parsers/cron_log.py

```diff
@@ -0,0 +1,39 @@
+"""
```

Comment:
@reidliu41 please add the file `docs/shared_parsers_catalog/cron_log.rst` as documented [here](https://insights-core.readthedocs.io/en/latest/docs_guidelines.html#testing-your-docstring) to include your parser in the documentation build.